src/tests/test-data/notes.json

```diff
@@ -1,6 +1,6 @@
 [
   { 
-    "id": 1,
+    "id": 60,
```

Comment:
why modify this data?

Reply:
Because POST by default make primary key = 1 in database and we have 500 error. So that is why we change id to another number.

Reply:
Yeah, by the same reason i've modified existing testcase for GET note/resolve-hostname/:hostname "Returns note for specified hostname"